src/utils/BackofficeAuth.ts

```diff
@@ -0,0 +1,9 @@
+import { backofficeAuthFactory } from '@payhawk/external-integration-service-utils';
+
+import { config } from '../Config';
+
+const credentials: { [user: string]: string } = Object.freeze({
+    [config.backOffice.username]: config.backOffice?.password,
```

Comment:
seems like it shouldn't be undefined ?

```suggestion
    [config.backOffice.username]: config.backOffice.password,
```